### DISCUSSION TITLE
Can I Remove the Camera Icons in iTwins Viewer

### DISCUSSION THREAD

**Discussion by sweetld** [iTwin/itwinjs-core#3580] [Q&A] [answered]
Hi,

I have been asked if it's possible to remove the Camera Icons (that appear in the right hand side of the iTwins Viewer UI) can be removed...

I have turned off a couple of other icons in the iTwins Viewer by changing the configuration passed to the iTwins Viewer, however I have not seen anything relating to these two camera icons.

Is this possible, have I just missed the settings, could someone please provide an example?

Thanks and regards

**Comment by Myzik-Dev** [comment]
With v3.x versions, you can remove the camera icon by customizing the UiItemsProvider.

**Reply by sweetld** [reply]
Thanks, unfortunately we are still using an older v2.x version...

I don't suppose that there is a way to do this in the v2.x version?

**Comment by aruniverse** [comment, marked as answer]
Take a look at the ui config in v2 https://github.com/iTwin/viewer/blob/master/packages/modules/viewer-react/src/types.ts#L194

**Reply by sweetld** [reply]
Thanks!!
